pkg/virt-handler/vm.go

```diff
@@ -689,6 +692,33 @@ func (d *VirtualMachineController) migrationSourceUpdateVMIStatus(origVMI *v1.Vi
 	return nil
 }
 
+type NetworkStatus struct {
+	Name      string   `yaml:"name"`
+	Ips       []string `yaml:"ips"`
+	Interface string   `yaml:"interface"`
+}
+
+// Using the downward API, look for dedicated migration network migration0 and, if found, set migration IP to its IP
+func findMigrationIP(migrationIp *string) error {
```

Comment:
I think we were going forth and back a little bit here on using the downwards api or directly reading the IP from the interface. What was the final reason for using the downward api (which is fine by me in general)?

Reply:
I'm pretty sure that was an offline suggestion from you, and it does look cleaner using the downwards API. Not sure if there's much more to it.

Reply:
Ack.